src/Modules/StatCan.OrchardCore.EmailTemplates/Startup.cs

```diff
@@ -48,6 +51,9 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddSingleton<IDeploymentStepFactory>(new DeploymentStepFactory<AllEmailTemplatesDeploymentStep>());
             services.AddScoped<IDisplayDriver<DeploymentStep>, AllTemplatesDeploymentStepDriver>();
 
+            services.AddActivity<EmailTemplateTask, EmailTemplateTaskDisplayDriver>();
```

Comment:
@arnaudroystatcan I just noticed this after merging it. This should go into a seperate Startup that depends on the Workflows feature.